ThunderCatsTests/UICollectionReusableViewTests.m

```diff
@@ -0,0 +1,36 @@
+//
+//  UICollectionReusableViewTests.m
+//  ThunderCats
+//
+//  Created by Abbey Jackson on 2016-05-12.
+//  Copyright © 2016 Metova. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "UICollectionReusableView+TCAdditions.h"
+
+@interface UICollectionReusableViewTests : XCTestCase
+
+@end
+
+@implementation UICollectionReusableViewTests
+
+- (void)testReuseIdentifier
+{
+    NSString *identifier = [UICollectionReusableView tc_reuseIdentifier];
+    XCTAssertEqualObjects(identifier, @"UICollectionReusableView");
```

Comment:
This test would easily pass even if this method just returned this raw string, but the beauty of this method is that it is using unique strings for every class (using the class name).  To really test this, we should have at least one class defined in this implementation file that is a subclass of the reusable view and assert this same thing also works for that subclass.